testsuite/features/step_definitions/smdba_steps.rb

```diff
@@ -65,7 +65,7 @@
 end
 
 Then(/^I issue command "(.*?)"$/) do |command|
-  $output, _code = $server.run(command, check_errors: false)
+  $output, _code = $server.run(command, check_errors: true)
```

Comment:
looks good to me, just don't forget to update the year at the top of the file

Reply:
This step only collect the logs, so the next step have an assert that it suppose to throw the error.

Reply:
Good point, some have output checking afterwards, others don't, it might make more sense to add some steps to check the output in those that don't have any checking (e.g.  https://github.com/uyuni-project/uyuni/blob/master/testsuite/features/finishing/srv_smdba.feature#L88)

Reply:
Personally, I would refactor all these steps that use this global variable storing the output.
This is impossible to handle it as it is when we start parallelizing features.
Instead, we should have a global variable (as feature scope is not supported) and have a hashmap/dictionary on it, and we have a key_value pair, where in the key we store the feature filename, and in the value we store another map where we can store stuff for the feature, like for instance this "output" variable.

Reply:
That specific feature might be an exception to parallelization since it is being run in the finishing stage since it `is destructive for other features` ( [quote](https://github.com/uyuni-project/uyuni/blob/master/testsuite/run_sets/finishing.yml#L12) ) and starting to implement this hashmap dictionary might require its own card.

Reply:
Sure, sure :)